Library/Formula/cryptol.rb

```diff
@@ -0,0 +1,42 @@
+require "language/haskell"
+
+class Cryptol < Formula
+  include Language::Haskell::Cabal
+
+  homepage "http://www.cryptol.net/"
+  url "https://github.com/GaloisInc/cryptol/archive/v2.2.0.tar.gz"
+  sha256 "90d2cbe35db8b2a9fcd78eaa2c08ab0cd81641a30949ab855dde71d17429d3ee"
+  head "https://github.com/GaloisInc/cryptol.git"
+
+  depends_on "ghc" => :build
+  depends_on "cabal-install" => :build
+  depends_on "cvc4"
+
+  def install
+    cabal_sandbox do
+      if build.stable?
+        # When we're building from the stable tarball, we have to
+        # provide the hash ourselves. Update this when updating the
+        # stable release.
+        system "make", "REV=0536d0f", "PREFIX=#{prefix}", "install"
```

Comment:
The archive is just what Github provides from the tag, so in order to add that info we'd have to know the hash in advance of the commit. One option is to host a subsequent tarball elsewhere, but I'd rather not have different tarballs with different signatures for Homebrew vs. non. The other option is to not include a hash; everything will still build fine, it'll just not have hash info for bug reporting. If it's not possible to run `git rev-parse` or similar during the Homebrew build, I'd rather just maintain things as they are here.